lang/shared/src/main/scala/com/wavesplatform/lang/v1/traits/domain/APair.scala

```diff
@@ -0,0 +1,5 @@
+package com.wavesplatform.lang.v1.traits.domain
+
+import scodec.bits.ByteVector
+
+case class APair(amountAsset: Option[ByteVector], priceAsset: Option[ByteVector])
```

Comment:
Can this be `AssetPair` ?

Reply:
APair and AssetPair both is used in src/main/scala/com/wavesplatform/transaction/smart/RealTransactionWrapper.scala